newsfragments/49.feature.rst

```diff
@@ -0,0 +1 @@
+Enums for Ethereum networks provided by `ChainID` class.
```

Comment:
Same definition here as the description in the comment above.

> Add an `IntEnum` class defining EVM-compatible network name enums as their respective chain id `int` values.

Reply:
Oops, keep getting this backwards. Will double check in the future!